python/python/tests/test_remote_db.py

```diff
@@ -315,7 +315,7 @@ def handler(body):
         assert body == {
             "distance_type": "l2",
             "k": 10,
-            "prefilter": False,
```

Comment:
It appears the remote API was still defaulting to `prefilter=False`.  Is there any particular reason we wanted this behavior?

Reply:
The intention was to keep consistent with the local sync API. Looks like we switched the default in https://github.com/lancedb/lancedb/pull/2000, though. So we can switch it for remote.